rskj-core/src/test/java/co/rsk/peg/union/UnionBridgeIT.java

```diff
@@ -0,0 +1,996 @@
+package co.rsk.peg.union;
+
+import static co.rsk.peg.BridgeMethods.*;
+import static co.rsk.peg.BridgeMethods.GET_UNION_BRIDGE_CONTRACT_ADDRESS;
+import static co.rsk.peg.BridgeMethods.GET_UNION_BRIDGE_LOCKING_CAP;
+import static co.rsk.peg.BridgeMethods.INCREASE_UNION_BRIDGE_LOCKING_CAP;
+import static co.rsk.peg.BridgeMethods.RELEASE_UNION_BRIDGE_RBTC;
+import static co.rsk.peg.BridgeMethods.REQUEST_UNION_BRIDGE_RBTC;
+import static co.rsk.peg.BridgeMethods.SET_UNION_BRIDGE_CONTRACT_ADDRESS_FOR_TESTNET;
+import static org.ethereum.vm.PrecompiledContracts.BRIDGE_ADDR;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import co.rsk.core.Coin;
+import co.rsk.core.RskAddress;
+import co.rsk.peg.Bridge;
+import co.rsk.peg.BridgeIllegalArgumentException;
+import co.rsk.peg.BridgeSerializationUtils;
+import co.rsk.peg.BridgeStorageProvider;
+import co.rsk.peg.BridgeSupport;
+import co.rsk.peg.BridgeSupportTestUtil;
+import co.rsk.peg.constants.BridgeMainNetConstants;
+import co.rsk.peg.storage.BridgeStorageAccessorImpl;
+import co.rsk.peg.storage.StorageAccessor;
+import co.rsk.peg.union.constants.UnionBridgeConstants;
+import co.rsk.test.builders.BridgeBuilder;
+import co.rsk.test.builders.BridgeSupportBuilder;
+import co.rsk.test.builders.UnionBridgeSupportBuilder;
+import java.math.BigInteger;
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.TestUtils;
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ActivationConfigsForTest;
+import org.ethereum.core.CallTransaction;
+import org.ethereum.core.CallTransaction.Function;
+import org.ethereum.core.Repository;
+import org.ethereum.core.SignatureCache;
+import org.ethereum.core.Transaction;
+import org.ethereum.crypto.ECKey;
+import org.ethereum.vm.DataWord;
+import org.ethereum.vm.PrecompiledContracts;
+import org.ethereum.vm.exception.VMException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@TestInstance(Lifecycle.PER_CLASS)
+class UnionBridgeIT {
+
+    private static final ActivationConfig lovellActivations = ActivationConfigsForTest.lovell700();
+    private static final ActivationConfig allActivations = ActivationConfigsForTest.all();
```

Comment:
```suggestion
    private static final ActivationConfig.ForBlock lovellActivations = ActivationConfigsForTest.lovell700().forBlock(0L);
    private static final ActivationConfig.ForBlock allActivations = ActivationConfigsForTest.all().forBlock(0L);
```

wdyt? as we always do